gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/utils/GradlePluginTestBase.kt

```diff
@@ -15,12 +15,6 @@ abstract class GradlePluginTestBase {
     val defaultTestEnvironment: TestEnvironment
         get() = TestEnvironment(workingDir = testWorkDir)
 
-    val defaultAndroidxCompilerEnvironment: TestEnvironment
```

Comment:
I removed `defaultAndroidxCompilerEnvironment` and `compose.tests.androidx.compiler.version` because they are used only in one test, and it seems there is no reason to use it in another place in the future